### ISSUE TITLE
Add option to disable 8chans JS

### ISSUE THREAD

**Issue by ScriptKi77y** [ctrlcctrlv/infinity#197]
I'd like to request a simple option to disable execution of the sites main.js file. sort of how 4chan implemented some time ago.

This would really help in development of userscripts. The sites main.js file likes to break our 8chan-x script unless we intentionally add something to cause a runtime error with the sites jquery immediately on page load, but even that still leaves event handlers present and the sites autoupdate running and we'd like to avoid that if possible. Our script is a slightly more advanced version of what's running on the site with addition of possible error reporting and tested for compatibility with the majority of current browsers and mobile devices.

TL;DR 
we have our own version of main.js which we would prefer to use. Please add option to disable site default main.js Thank you.
##
